ksqldb-functional-tests/src/test/java/io/confluent/ksql/test/rest/RestTestExecutor.java

```diff
@@ -729,12 +725,23 @@ private static void threadYield() {
     }
   }
 
+  private static void verifyResponseFields(final HashMap<String, Integer> actualRecords, final HashMap<String, Integer> expectedRecords, final int idx){
```

Comment:
Just to understand: You count how many times you have seen each row in these maps?
If so, shouldn't you check that the maps have the same size and that also `expectedRecords` is contained in `actualRecords`?

Reply:
yes that assertion is still there, line 875: 

`      assertThat(
          "row count mismatch."
              + System.lineSeparator()
              + "Expected: "
              + expectedRows.stream()
              .map(Object::toString)
              .collect(Collectors.joining(INDENT, INDENT, ""))
              + System.lineSeparator()
              + "Got: "
              + rows.stream()
              .map(Object::toString)
              .collect(Collectors.joining(INDENT, INDENT, ""))
              + System.lineSeparator(),
          rows,
          hasSize(expectedRows.size())
      );`

Reply:
Thanks, added a description above.